File: go/src/github.com/eliudarudo/go-cli-app/questionnaire/input_mod.go
package questionnaire

import (
	"fmt"
	"strings"
)

// GetSelection function
func GetSelection() string {
	fmt.Println("==================================")
	fmt.Print("Enter your choice (Q to quit): ")

	var selection string
	fmt.Scanln(&selection)

	return strings.ToUpper(selection)
}

File: java/src/com/eliudarudo/cliapp/questionnaire/InputMod.java
package com.eliudarudo.cliapp.questionnaire;

import java.util.Scanner;

public class InputMod {
    public static String getSelection() {
        System.out.println("==================================");
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter your choice (Q to quit): ");

        try {
            String selection = scanner.nextLine().toUpperCase();
            return selection;
        } catch(Exception e) {
            return "";
        }
    }
}
